crates/containerd-shim-wasmtime/src/main.rs

```diff
@@ -1,6 +1,7 @@
-use containerd_shim_wasm::sandbox::cli::{revision, shim_main, version};
+use containerd_shim_wasm::sandbox::cli::{revision, shim_main_with_otel, version};
 use containerd_shim_wasmtime::WasmtimeInstance;
 
-fn main() {
-    shim_main::<WasmtimeInstance>("wasmtime", version!(), revision!(), "v1", None);
+#[tokio::main]
```

Comment:
Is this needed?  I thought we were injecting the async runtime into the `.install_batch(runtime::Tokio)`.  Similar to how we do it for the containerd client code

Reply:
Yes, unfortunately opentelemetry SDK in rust all require tokio runtime

Reply:
I am not familiar enough with rust async, will this cause any issues with shim or is it ok to sprinkle async in various areas?